src/gatsby-plugin-chakra-ui/theme.js

```diff
@@ -130,6 +130,13 @@ const customTheme = {
       textTransform: 'uppercase',
     },
   },
+  footer: {
+    background: '#001514',
```

Comment:
We can leverage the global color variable defined on line 38 in this file instead of defining the hex again!

Reply:
Ah completely missed that! Will sawp